worlds/dark_souls_3/docs/locations_en.md

```diff
@@ -2073,7 +2073,7 @@ static _Dark Souls III_ randomizer].
 <tr><td>RS: Great Swamp Ring - miniboss drop, by Farron Keep</td><td>Dropped by Greater Crab in Crucifixion Woods close to the Farron Keep outer wall</td></tr>
 <tr><td>RS: Green Blossom - by deep water</td><td>In the Crucifixion Woods crab area out in the open, close to the edge of the deep water area</td></tr>
 <tr><td>RS: Green Blossom - water beneath stronghold</td><td>In the Crucifixion Woods crab area close to the Crucifixion Woods bonfire, along the left wall of the water area, to the right of the entrance to the building before Crystal Sage</td></tr>
-<tr><td>RS: Heretic&#x27;s Staff - stronghold left room</td><td>In the building before Crystal Sage, entering from near Crucifixion Woods, in a corner under the first stairwell and balcony</td></tr>
+<tr><td>RS: Heretic&#x27;s Staff - stronghold left room</td><td>In the building before Crystal Sage, entering from near Cruficixion Woods, in a corner under the first stairwell and balcony</td></tr>
```

Comment:
We're adding the typo back in?

Reply:
These typos are coming from the upstream descriptions. I've fixed these two upstream so now they shouldn't be regenerated. (I've also fixed the Unicode encoding to avoid weird Siegbrau characters).